docs/topics/modules/committing_pushing_changes_git.adoc

```diff
@@ -17,27 +19,22 @@ endif::[]
 [discrete]
 == Procedure
 
-. In the workspace browser tab, click *Git* from the menu options and select *Commit* from the displayed options.
-+
-image::commit_menu.png[Git Commit Menu]
-+
+. In the workspace browser tab, click *Git* from the menu bar options and select *Commit* from the displayed options.
+
 . In the *Commit to repository* dialog box:
 
-.. Select all the changed and new files to commit.
+.. Select all the changed and new files to add them to the commit.
 .. Add a commit message describing your changes.
 .. Select the *Push committed changes to:* check box.
 .. Click btn:[Commit].
-+
-image::{context}_commit_dialog.png[Commit changes]
-+
+
 When the commit succeeds, the following message displays:
 +
 image::pushed_to_origin.png[Pushed to origin]
-+
+
 You have now committed your code changes to GitHub.
 
-+
 //for hello world
 ifeval::["{context}" == "hello-world"]
-. You have now completed the task, *Modify the quickstart codebase*, in the *Test Iteration*. Ensure that you change the state of the work item to *Closed* using the *Plan* tab.
+You have now completed the task, *Modify the quickstart codebase*, in the *Test Iteration*. Change the state of the work item to *Closed* using the *Plan* tab.
 endif::[]
```

Comment:
@mishaone this section was made into a step after you pointed out that it seemed like an instruction. Are we going back to what I originally wrote? Personally, I prefer it like this, as I had originally written. Just wanted to get a confirmation so that we uniformly follow this in other modules too.

Reply:
No, sorry, Preeti, this is purely for not breaking the rendering. One of the changes I did a bit earlier in this module meant we had a stray + appearing in the rendered text just before the conditional text so that the step numbering would continue. This was just consistently breaking so I removed the + and made the step a paragraph instead. We should probably find a better way to add conditional steps and get them playing well together, but at the time it just seemed like a lot of aggravation for a minor change.

Reply:
No problem Misha, in that case I will work on it and change it back to numbered step without affecting the rendering, while working on my modules just so that we are consistent in the doc.